consensus/defs.c

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "consensus/defs.h"
+
+flex_trit_t genesis_hash[FLEX_TRIT_SIZE_243] = {0};
```

Comment:
No globals please? If need be, let's make this a #define, but I'd much rather see this live in some state object